src/replication_handler/include/replication_handler/replication_handler.hpp

```diff
@@ -51,8 +51,7 @@ void SystemRestore(replication::ReplicationClient &client, system::System &syste
       [](auto &state) { return state != memgraph::replication::ReplicationClient::State::RECOVERY; });
   {
     using enum memgraph::flags::Experiments;
-    bool full_system_replication =
-        flags::AreExperimentsEnabled(SYSTEM_REPLICATION) && license::global_license_checker.IsEnterpriseValidFast();
+    bool full_system_replication = license::global_license_checker.IsEnterpriseValidFast();
```

Comment:
This is now inconsistent because there is no such a thing as `fully_system_replication`, there is only `replication`, just remove the variable and replace it with `license::global_license_checker.IsEnterpriseValidFast()`.

Reply:
I was thinking of doing a cleanup, I just want to see if all the tests pass